[CUBRIDMAN-222]  매뉴얼 설명 수정  및 예제 수정

bagus-kim opened the pull request:
http://jira.cubrid.org/browse/CUBRIDMAN-222

1. RANDOM, RAND설명 수정요청에 따른 수정작업 및 예제 수정 작업

ASIS : RANDOM / RAND 함수는 0 이상 2 31 미만 구간에서 임의의 정수 값을 반환하며...
TOBE : RANDOM / RAND 함수는 0 이상 2^31 미만 구간에서 임의의 정수 값을 반환하며....

2. 예약어 목록에 'ACCESS' 추가
3. 시스템 설정의 목록 추가 
  3-1) HA설정, SHARD설정 추가
  3-2) ;set에 대한 link추가
4. block_ddl_statement에 "yes로 설정된 경우 통계 정보 갱신 구문(update statistics)도 제한될 수 있음을 유의하여야 한다." 설명 추가
5. 디렉토리 -> 디렉터리로 단어 수정
6. server_stmt.rst에서 :ref:`cci_connect_with_url` link오류 수정
7. IS NULL연산자 예제 수정 : CREATE INDEX idx ON t (a) WHERE (not a) IS NULL; -> CREATE INDEX idx ON ta (a) WHERE (not ta.a<>0 ) IS NULL;
8. 예제문 수정 : ALTER TABLE t_def a a int; -> ALTER TABLE t_def MODIFY a int;

bagus-kim committed:
RAND()범위 (2 31미만 -> 2^31미만)으로 내용 수정

bagus-kim committed:
가독성을 위한 sql 예제문 수정

bagus-kim committed:
국문 rst오류 수정 / 영문 예제수정

bagus-kim committed:
http://www.cubrid.com:8888/browse/RND-2290
예약어 목록에 'ACCESS'추가

bagus-kim committed:
국문파일 누락되어 commit
http://www.cubrid.com:8888/browse/RND-2290 예약어 목록에 'ACCESS'추가

bagus-kim committed:
http://www.cubrid.com:8888/browse/RND-2316
1. 시스템 설정 목록 수정
2. ;set에 명령어 link처리

bagus-kim committed:
[CUBRIDMAN-222] http://www.cubrid.com:8888/browse/RND-2291
block_ddl_statement가 yes인경우 통계 정보 갱신 에러가 발생할 있다는 내용 추가

hiclass commented:
기존 매뉴얼처럼 불필요 부분 삭제하고 예제가 작성 되었으면 합니다.
삭제 필요, csql>, === <Result of SELECT Command in Line 1> ===, 및 기타 빈 줄 등

bagus-kim committed:
csql> 및 불필요한 문자열 삭제

bagus-kim committed:
**block_ddl_statement** 에서 no --> **no**로 수정

bagus-kim reviewed (commented):
csql>, === <Result of SELECT Command in Line 1> ===
삭제 처리되었습니다.

bagus-kim committed:
http://www.cubrid.com:8888/browse/RND-2234

단어수정 : 디렉토리 -> 디렉터리

bagus-kim committed:
server_stmt.rst에서 :ref:`cci_connect_with_url` link오류 수정

bagus-kim committed:
http://www.cubrid.com:8888/browse/RND-2336
IS NULL연산자 예제 수정 : CREATE INDEX idx ON t (a) WHERE (not a) IS NULL; -> CREATE INDEX idx ON ta (a) WHERE (not ta.a<>0 ) IS NULL;

bagus-kim committed:
http://www.cubrid.com:8888/browse/RND-2309
예제문 수정 : ALTER TABLE t_def a a int; -> ALTER TABLE t_def MODIFY a int;
